gpytorch/priors/gp_prior.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+from .torch_priors import MultivariateNormalPrior
+
+
+class GaussianProcessPrior(MultivariateNormalPrior):
```

Comment:
I don't love having this prior as a public class - since its really specific to FKL and its name might cause confusion. Could you put it as a private class in the SpectralGPKernel file?